packages/plugins/docs-ui/src/lib/pages/browse/docs-browse-page.component.html

```
@@ -59,9 +59,22 @@ <h4 class="docs-browse-title">{{ 'DOCS.TITLE' | translate }}</h4>
 				</button>
 			</fieldset>
 			<ng-template [ngxPermissionsOnly]="[permissions.DOCS_CREATE]">
-				<button nbButton size="small" appearance="outline" (click)="openNewPageDialog()">
-					<nb-icon icon="file-add-outline"></nb-icon>
-					{{ 'DOCS.TREE.NEW_PAGE' | translate }}
+				<!-- `New ▾` (`01-ux-spec.md` §2): Folder | Page. A folder was previously
+				     unreachable from here, so a brand-new organization — whose tree is empty and
+				     whose only other create affordance is a node context menu that needs an
+				     existing node — had no way to create its first folder at all. -->
+				<button
+					#newMenuTrigger
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)